syntaxes/tests/vba/metadata.bas

```diff
@@ -12,10 +12,20 @@ END
  End
 '^^^ keyword.control.vba
 
-Sub test()
+Sub Begin(ByVal MyString As String)
 

```

Comment:
Please add the scope check here too:

`   '^^^^^ - keyword.metadata.vba`

Reply:
Done